src/webgpu/shader/execution/expression/call/builtin/texture_utils.ts

```diff
@@ -718,6 +726,37 @@ function zeroValuePerTexelComponent(components: TexelComponent[]) {
   return out;
 }
 
+const kSamplerFns: Record<GPUCompareFunction, (ref: number, v: number) => boolean> = {
+  never: (ref: number, v: number) => false,
+  less: (ref: number, v: number) => ref < v,
+  equal: (ref: number, v: number) => ref === v,
+  'less-equal': (ref: number, v: number) => ref <= v,
+  greater: (ref: number, v: number) => ref > v,
+  'not-equal': (ref: number, v: number) => ref !== v,
+  'greater-equal': (ref: number, v: number) => ref >= v,
+  always: (ref: number, v: number) => true,
+} as const;
+
+function postProcess<T extends Dimensionality>(
```

Comment:
Is the intention to use this for other post processing as well? Otherwise, maybe this could just be named something related to `Compare` since that's all it's doing (I think?)

Reply:
renamed to `applyCompare`